001-devcycle.conf.template

```diff
@@ -0,0 +1,25 @@
+WSGIPythonPath usr/local/devcycle
+
+<VirtualHost *:80>
+        ServerName devcycle.se.rit.edu
+        ServerAlias devcycle.se.rit.edu
+        ServerAdmin someaddress@example.com
+
```

Comment:
If this is a template file, I would like to see these values more visibly as they would need to change.  Something like

```
ServerName SERVER_HERE
ServerAlias SERVER_HERE
ServerAdmin EMAIL_HERE
```